applications/asset_tracker_v2/doc/sensor_module.rst

```diff
@@ -7,7 +7,7 @@ Sensor module
    :local:
    :depth: 2
 
-The sensor module interacts with external sensors present on the `Thingy:91 <Thingy:91 product page>`_.
+The sensor module interacts with external sensors present on the `Thingy:91 <Thingy\:91 product page_>`_.
```

Comment:
Was the `\` added on purpose?

Reply:
That must be just a mishap. No backslash needed there.

Reply:
The link don't work with the combination of these special characters, and adding \ would fix the issue.
May be to override the special meaning provided to these markup characters.
https://docutils.sourceforge.io/docs/user/rst/quickref.html#escaping

Reply:
Oh, good point Divya. I missed that.